src/config.inc

```diff
@@ -76,7 +76,7 @@ endif
 OSX_IDENTITY="Developer ID Application: Daniel Kroening"
 
 # Detailed version information
-CBMC_VERSION = 5.25.0
+CBMC_VERSION = 5.24.0
```

Comment:
Please rebase on to the latest version.  This PR cannot be merged while it reverts the version number.